numbers_letters_and_variables/number_of_seconds.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+
+number_of_seconds_per_minute = 60
+number_of_minutes_per_hour = 60
+number_of_seconds_per_hour = number_of_minutes_per_hour * number_of_seconds_per_minute
+number_of_hours_per_day = 24
+number_of_seconds_per_day = number_of_hours_per_day * number_of_minutes_per_hour * number_of_seconds_per_minute
```

Comment:
You could use `number_of_seconds_per_day = number_of_hours_per_day * number_of_seconds_per_hour`